modules/markup/html.go

```diff
@@ -351,6 +351,46 @@ func (ctx *postProcessCtx) visitNode(node *html.Node, visitText bool) {
 			visitText = false
 		} else if node.Data == "code" || node.Data == "pre" {
 			return
+		} else if node.Data == "i" {
+			for _, attr := range node.Attr {
+				if attr.Key != "class" {
+					continue
+				}
+				// Got a class
+				from := 0
+
+				for idx := strings.Index(attr.Val[from:], "icon") + from; idx >= from; {
```

Comment:
Why not `strings.Split()` or even a regexp? Seems overly complicated and error prone. In fact, I think the condition `idx >= from` is not right; it will be probably false for the second word:
```
val := "not-icon icon"
from := 8 // space after "not-icon"
idx := strings.Index(val[from:], "icon")
// val[from:] ==> " icon"
// idx ==> 1
// idx < from!
```